lib/routes/tencent/wechat/tgchannel.js

```diff
@@ -17,9 +18,15 @@ module.exports = async (ctx) => {
                 let title = item.find('.tgme_widget_message_text > a:nth-child(5)').text() || item.find('.tgme_widget_message_text > a:nth-child(2)').text();
 
                 const all_text = item.find('.tgme_widget_message_text').text();
-                if (all_text.indexOf(':') !== -1) {
-                    author = all_text.split(':')[0].split(' ')[1];
-                    title = author + ': ' + title;
+                const spaceIndex = all_text.indexOf(' ');
+                const colonIndex = all_text.indexOf(':');
+                if (spaceIndex !== -1 && colonIndex !== -1) {
+                    author = all_text.slice(spaceIndex + 1, colonIndex);
+                    if (mpName && author !== mpName) {
+                        return Promise.resolve(null);
```

Comment:
```suggestion
                        return;
```

这里 return null 有什么标准吗？
async 的返回值就是 promise，为什么包裹一层

Reply:
在路由填入 `mpName` 开启了公众号名筛选时，就会判断 `author`（在消息中提取的公众号名）是不是与 `mpName` 的一致，不一致就 return，不再继续后面的流程；在输出时会被 filter 掉，就达到了筛选的效果。